### PR TITLE
FF85 Devtools>Page Inspector: Can now toggle :focus_visible

### LINKED ISSUE #296: [DevTools] Allow Firefox devtools to toggle the focus-visible pseudo-class
## Acceptance criteria

1. - [x] The listed features are documented sufficiently on MDN
2. - [x] BCD is updated - NA
3. - [x] Interactive example and data repos are updated if appropriate - NA
4. - [ ] The content has been reviewed as needed

## Features to document

1. Firefox-only — Allow devtools to toggle the focus-visible pseudo-class

## Related Gecko bugs

1. - [ ] [Allow devtools to toggle the focus-visible pseudo-class.](https://bugzilla.mozilla.org/show_bug.cgi?id=1617608)

For folks helping with Firefox-related documentation features — make sure above AC have been done, but also:

* Set bugs to dev-doc-complete
* Add entry to Firefox release notes if feature is enabled in release
* Add entry to Firefox experimental features page if feature is not yet enabled in release

### PR DESCRIPTION
**Pull request by hamishwillee**
Fixes #296

This is an update for Firefox 85 release
- Updates images and docs to refer to` :focus_visible`
- Updates release notes for FF85

Supersedes/replaces #759

### CODE REVIEW

**Comment by hamishwillee** [comment]
@chrisdavidmills This is failing in tests because it says I should shrink the new files. But it isn't clear how I would do this/what dependencies are required. Do you know how to do this?

I tried  yarn filecheck  'files/en-us/tools/page_inspector/how_to/examine_and_edit_css/hover_indicators.png'  locally

**Comment by chrisdavidmills** [comment]
@hamishwillee I got caught out by this once before, so I discussed it with the engineers, they improved the error messages, and I documented what to do: https://github.com/mdn/content#compressing-images

Also, below where it says "Some checks were not successful", if you click the "Details" link to the right of the failing test, you'll see the test output, and it contains a line telling you exactly what command to enter to do the compression in your local repo.

**Comment by hamishwillee** [comment]
Hi @chrisdavidmills ,

Sorry, I had tried that but above only showed the version without the `--save-compression` option - and didn't put sufficient debug information for you to understand that :-(
```
yarn filecheck 'files/en-us/tools/page_inspector/how_to/examine_and_edit_css/hover_indicators.png
```
Either case gives same error on windows as shown below:

```
D:\github\mozilla\mdn_content\content>yarn filecheck "files/en-us/tools/page_inspector/how_to/examine_and_edit_css/show_pseudo_classes_hover.png" --save-compression
yarn run v1.22.5
$ env-cmd --silent cross-env CONTENT_ROOT=files node node_modules/@mdn/yari/filecheck/cli.js --cwd=$(pwd) files/en-us/tools/page_inspector/how_to/examine_and_edit_css/show_pseudo_classes_hover.png --save-compression
files/en-us/tools/page_inspector/how_to/examine_and_edit_css/show_pseudo_classes_hover.png was unexpected at this time.

D:\github\mozilla\mdn_content\content>  "D:\github\mozilla\mdn_content\content\node_modules\.bin\\node.exe"  "D:\github\mozilla\mdn_content\content\node_modules\.bin\\..\env-cmd\bin\env-cmd.js" --silent cross-env CONTENT_ROOT=files node node_modules/@mdn/yari/filecheck/cli.js --cwd=$(pwd) files/en-us/tools/page_inspector/how_to/examine_and_edit_css/show_pseudo_classes_hover.png --save-compression
error Command failed with exit code 255.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
Note:
- Same error without the `--save-compression` flag.
- Have updated to the master branch and re-run `yarn install`
- Have this version of node
  ```
  D:\github\mozilla\mdn_content\content>node -v
  v12.18.1
   ```
- Looked at `info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.` but nothing obviously related.

So basically I was wondering if I need a particular version of node or to install anything else.

**Comment by hamishwillee** [comment]
@chrisdavidmills Further to above, it does work with same branch on Ubuntu (with node 12.20.0 installed). I have pushed the fixes. However it would still be good to understand how I can get this to work on Windows.  I've kept this branch for testing https://github.com/hamishwillee/content/tree/backup_devtools_badcomp

**Review by chrisdavidmills — APPROVED** [review]

**Comment by chrisdavidmills** [comment]
I've got this merged now at least ;-)

I've got no idea how to make it work on Windows. Maybe raise a separate issue on the yari repo to explain the aituation and ask?
